js/components/url.js

```diff
@@ -9,11 +9,22 @@ Fliplet.FormBuilder.field('url', {
       type: String
     }
   },
+  data: function() {
+    return {
+      rules: {
+        urlCase: new RegExp(/^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@%!\$&'\*\+,;=.]+$/i)
+      }
+    };
+  },
   validations: function() {
     var rules = {
       value: {
         // URL regex taken form https://www.regextester.com/94502 and added % sign
```

Comment:
This comment should be moved to where the regexp is defined

Reply:
Done